src/components/UserPortal/CreateDirectChat/CreateDirectChat.tsx

```diff
@@ -62,9 +61,9 @@ export default function createDirectChatModal({
   createDirectChatModalisOpen,
   chatsListRefetch,
 }: InterfaceCreateDirectChatProps): JSX.Element {
-  const { t } = useTranslation('translation', {
-    keyPrefix: 'userChat',
-  });
+  // const { t } = useTranslation('translation', {
+  //   keyPrefix: 'userChat',
+  // });
```

Comment:
Please remove these lines
```suggestion

```

Reply:
Yes, I removed it. 
![image](https://github.com/user-attachments/assets/6deb125a-7a6a-41ba-a8ff-983a8019b771)